apps/src/blockly/addons/workspaceSvgFrame.js

```diff
@@ -108,10 +108,16 @@ export default class WorkspaceSvgFrame extends SvgFrame {
 
   getFrameX() {
     // In LTR the svg should be to the right of the toolbox, plus a margin.
+    const metricsManager = this.element_.getMetricsManager();
     let frameX = frameSizes.MARGIN_SIDE / 2;
-    const toolbox = this.element_.toolbox_;
-    if (toolbox) {
-      frameX += toolbox.width_;
+    // Toolbox width > 0 if we have a categorized toolbox.
+    const toolboxWidth = metricsManager.getToolboxMetrics().width;
```

Comment:
This change is here in preparation for [CT-217](https://codedotorg.atlassian.net/browse/CT-217), to support resizing the svg if there's an uncategorized toolbox. FYI @mikeharv